projects/plugins/launchpad/src/js/components/Admin.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { AdminPage, AdminSection, AdminSectionHero } from '@automattic/jetpack-components';
+
+/**
+ * Internal dependencies
+ */
+import useConnection from '../hooks/useConnection';
+import './admin-style.scss';
+
+/* eslint react/react-in-jsx-scope: 0 */
+const Admin = () => {
+	const [ connectionStatus, renderConnectScreen, renderConnectionStatusCard ] = useConnection();
+	const [ connectionLoaded, setConnectionLoaded ] = useState( false );
+
+	useEffect( () => {
+		if ( 0 < Object.keys( connectionStatus ).length ) {
+			setConnectionLoaded( true );
+		}
```

Comment:
I suggest to create an utility function outside the component, that receives the `connectionStatus` and return if connection is loaded, and reuse it here and state creation, this will avoid a unnecessary re-render at first render (which could cause some blink at the screen, since it will load at one state and update to another).

```javascript
const connectionIsLoaded = connectionStatus => Object.keys( connectionStatus ).length >=1
const Admin = () => {
  const [ connectionLoaded, setConnectionLoaded ] = useState( connectionIsLoaded(connectionStatus) );
  useEffect( () => {
    setConnectionLoaded( connectionIsLoaded(connectionStatus) );
  }, [connectionStatus])
}
```

Reply:
Fixed in bd97049